metrics/metrics.go

```diff
@@ -111,7 +111,7 @@ func RegisterMetrics() {
 	prometheus.MustRegister(DbStmtNodeCounter)
 	prometheus.MustRegister(StoreQueryFeedbackCounter)
 	prometheus.MustRegister(TiKVBackoffCounter)
-	prometheus.MustRegister(TiKVBackoffHistogram)
+	prometheus.MustRegister(TiKVBackoffDuration)
```

Comment:
It's better to follow the existing variable naming rule: `s/TiKVBackoffDuration/TiKVBackoffDurationHistogram/`